mojave/items/guns/projectiles/energy-proj.dm

```diff
@@ -42,6 +42,14 @@
 	wound_bonus = 0
 	bare_wound_bonus = 5
 
+/obj/projectile/beam/ms13/laser/wattz_pistol/magneto
+	name = "laser beam"
+	range = 25
+	damage = 15
+	subtractible_armour_penetration = 20
+	wound_bonus = 0
```

Comment:
This is a redundant extra definition. The damage is too technically but that's good to have redefined just in case.

Reply:
fixed the redundancy